### ISSUE TITLE
Consecutive doxygen builds are not identical

### ISSUE THREAD

**Issue by aniketsalve22** [doxygen/doxygen#9838]
**Describe the bug**
After doing consecutive doxygen builds, even if no changes are done to any source file, the output HTML file have formatting changes.
To be specific the </div> tag after "iframe" tag changes its position.

**Expected behavior**
If no changes are done in the source file, the output consecutive doxygen builds should be identical. 
After comparing the outputs (all generated HTML files) should be identical. The HTML file format should also be identical.
 
**Screenshots**
After 1st build
![1st_build](https://user-images.githubusercontent.com/14127049/217024096-6ac07ab0-dbb6-4227-a504-7cb213980478.png)

After 2nd build
![2nd_build](https://user-images.githubusercontent.com/14127049/217024159-b378b34b-ee83-466d-8cb7-5739d434de5c.png)

The position of </div> tag is changed

**To Reproduce**
* Make sure to clean the old outputs.

* Set following attributes in doxyfile
HAVE_DOT = YES
DOT_IMAGE_FORMAT = svg

* Do a doxygen build (for example run doxygen demo/demo.doxyfile)
* Create a copy the build output and do one more doxygen build
* Compare both the build outputs for difference in HTML files (may be the class html files)

NOTE: If we set HAVE_DOT=NO or DOT_IMAGE_FORMAT=png then the outputs are identical.

**Version**
Doxygen version: 1.9.5 (2f6875a5ca481a69a6f32650c77a667f87d25e88)
Platform: Linux 11

**Additional context**
We are trying to implement incremental behavior for [doxysphinx](https://boschglobal.github.io/doxysphinx/index.html). And in order to compare the build outputs of doxygen, it should always stay identical if no input files are changed.

**Comment by albert-github** [comment]
There is no example with the issue so it is a bit hard / not possible to reproduce the problem as well as that you refer to an unknown file `demo/demo.doxyfile`, so
- Can you please attach a, small, self contained example (source+configuration file in a, compressed, tar or zip file) that allows us to reproduce the problem? Please don't add external links as they might not be persistent.


You use the 1.9.5 version of doxygen and when I remember correctly I've seen the same / similar behavior as well and this has been fixed but I'm not sure whether it has been fixed in the 1.9.6 version or in the current master (1.9.7 (e1442541e81ac04cbc8386be841a70a14e6d833a). Please try with the mentioned versions.

**Comment by aniketsalve22** [comment]
[doxygen.zip](https://github.com/doxygen/doxygen/files/10741943/doxygen.zip)

I have added an example to reproduce the issue (using the doxygen version 1.9.6)
Please take a look into docs folder as I have already zipped two sample build outputs. 
Compare line number 60 and 65 in following files to see the change in </div> position of </div> tag
* docs\html_1st_build\classdoxysphinx_1_1rst_1_1Car.html
* docs\html_2nd_build\classdoxysphinx_1_1rst_1_1Car.html

To reproduce the issue easily:

1. Unzip folder and open in VS Code editor (install [Dev Container](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension)
2. Press `ctrl + shift + p` and type ` Dev Containers: Reopen in Container` (Wait for VS Code to open the code in container)
3.  Open terminal and run command `doxygen demo/demo.doxyfile`  
4.  Create a copy of 'docs\html' folder
5. Again run the command `doxygen demo/demo.doxyfile` in terminal
6.  Compare the docs\html folder with the copied folder to see the difference in `classdoxysphinx_1_1rst_1_1Car.html` file

**Comment by albert-github** [comment]
Some files referred in the settings file are missing but this is of no importance to the problem, also the usage of the vs code editor is not of importance as we can directly start doxygen with the mentioned settings file.
The differences are of the form:
```
< <div class="center"><iframe scrolling="no" frameborder="0" src="classdoxysphinx_1_1rst_1_1Car__coll__graph.svg" width="415" height="80"><p><b>This browser is not able to show SVG: try Firefox, Chrome, Safari, or Opera instead.</b></p></iframe>
< </div>
---
> <div class="center"><iframe scrolling="no" frameborder="0" src="classdoxysphinx_1_1rst_1_1Car__coll__graph.svg" width="415" height="80"><p><b>This browser is not able to show SVG: try Firefox, Chrome, Safari, or Opera instead.</b></p></iframe></div>
```
i.e. the placing of the `</div>` tag.

As expected:
- This problem was identified and fixed by means of #9785 
- When using the current master (1.9.7 (a72c7b6629eb7675939cdfecf9eb21473bf62e73) the problem does not appear anymore.

Problem is fixed but not released.
